api/swagger-spec/v1beta3.json

```diff
@@ -7276,38 +7483,10 @@
      }
     }
    },
-   "v1beta3.ServicePort": {
-    "id": "v1beta3.ServicePort",
-    "required": [
-     "name",
-     "protocol",
-     "port",
-     "targetPort"
-    ],
-    "properties": {
-     "name": {
-      "type": "string",
-      "description": "the name of this port; optional if only one port is defined"
-     },
-     "port": {
-      "type": "integer",
-      "format": "int32",
-      "description": "the port number that is exposed"
-     },
-     "protocol": {
-      "$ref": "v1beta3.Protocol",
-      "description": "the protocol used by this port; must be UDP or TCP; TCP if unspecified"
-     },
-     "targetPort": {
-      "type": "string",
-      "description": "the port to access on the pods targeted by the service; defaults to the service port"
-     }
-    }
-   },
    "v1beta3.ServiceSpec": {
     "id": "v1beta3.ServiceSpec",
     "required": [
-     "ports",
+     "port",
```

Comment:
Something's not right here. This is reverting multi-port services, among other things.